src/modules/configurator/components/ConfigurationsEmpty.tsx

```diff
@@ -13,7 +14,7 @@ export function ConfigurationsEmpty() {
         <p className="text-center par-font-size">
           You haven't configured any cars yet. You may choose to{" "}
           <Link
-            to="/home/configure-a-car/car-select"
+            to={`${configuratorRoutes.configureACar}/car-select`}
```

Comment:
cat-select can also be a route it there are no dynamic params

Reply:
Added car-select to `configuratorRoutes`.